packages/lesswrong/server/editor/make_editable_callbacks.ts

```diff
@@ -98,13 +108,14 @@ function addEditableCallbacks<N extends CollectionNameString>({collection, optio
   const collectionName = collection.collectionName;
 
   getCollectionHooks(collectionName).createBefore.add(
-    async function editorSerializationBeforeCreate (doc: AnyBecauseTodo, { currentUser, context }: AnyBecauseTodo)
+    async function editorSerializationBeforeCreate (doc: DbType, {currentUser, context})
   {
```

Comment:
```suggestion
    async function editorSerializationBeforeCreate (doc: DbType, {currentUser, context}) {
```

Remember when Jim thought that newline curly braces were the thing?